kubernetes/retool-postgres.yaml

```diff
@@ -52,7 +52,7 @@ spec:
               name: retoolsecrets
               key: postgres_password
         name: postgres
-        image: postgres:11.12
+        image: postgres:11.13
```

Comment:
match the helm charts: https://github.com/tryretool/retool-helm/blob/main/values.yaml#L108